app/helpers/application_helper.rb

```diff
@@ -1,8 +1,8 @@
 module ApplicationHelper
-  def header_nav
+  def header_nav(signed_in)
     navPresenter = NavigationPresenter.new
     
-    navPresenter.nav_header
+    navPresenter.nav_header(signed_in)
```

Comment:
Not sure if this is a convention thing, but is it worth having the methods in the helper and presenter named the same, rather than reverse versions of each other? Or in some differential way, other than a straight reverse of each other

Reply:
changed so method names match